### PR TITLE
Make hamburger menu button responsive

### PR DESCRIPTION
**Pull request by lilianaweimer**
- Driver/navigator (Liliana driving, Jenna & Alex navigating)
- We made the hamburger menu button toggle between the hamburger and X
- Deleted an unused function

- We need to clean up our JS/CSS a bit

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,31 +3,18 @@ var hamburgerMenu = document.querySelector(".hamburger-menu");
...
-      image.src = "images/menu.svg";
-      filterStarred.style.display = "none";
-  } else {
-      filterStarred.style.display = "block";
-  }
+  // var menuButton = hamburgerImage.src;
+  // if (menuButton === "images/menu.svg") {
+  //   console.log('test!');
+  // }
+  hamburgerImage.src = (hamburgerImage.src.match("images/menu-close.svg")) ? "images/menu.svg" : "images/menu-close.svg";
```
Comment (jennadilullo8) [inline src/main.js:17]: It's amazing what the ternary operator can do. At first starting out with a conditional and now we did this. So much easier!

**Comment by jennadilullo8** [comment]
This was hard at first, but we got it!

File: src/main.js (hunk 1 above)
Comment (asodemann18) [inline src/main.js:17]: Nice use of the ternary operator!

**Review by asodemann18 — COMMENTED** [review]
Looks great!

**Pull request merged**